tensorflow/core/platform/default/logging.h

```diff
@@ -477,15 +477,22 @@ class TFLogEntry {
   }
 
  public:
-  explicit TFLogEntry(int severity, absl::string_view log_line)
-      : severity_(AsAbslLogSeverity(severity)), log_line_(log_line) {}
+  TFLogEntry(int severity, absl::string_view message)
+      : severity_(AsAbslLogSeverity(severity)), message_(message) {}
 
-  absl::LogSeverity log_severity() const { return severity_; }
-  std::string ToString() const { return std::string(log_line_); }
+  TFLogEntry(int severity, absl::string_view fname, int line, absl::string_view message)
+      : severity_(AsAbslLogSeverity(severity)), fname_(fname), line_(line), message_(message) {}
+
+  absl::LogSeverity Severity() const { return severity_; }
+  const std::string& FName() const { return fname_; }
+  int Line() const { return line_; }
+  const std::string& Message() const { return message_; }
 
  private:
   const absl::LogSeverity severity_;
-  const absl::string_view log_line_;
+  const std::string fname_;
+  int line_ = 0;
```

Comment:
nit: let's use -1 as it's more common as an invalid state.

Reply:
Will do